index.js

```diff
@@ -13,21 +13,21 @@ const refs = {
 
 const SKIP = {
   4: ['optional/zeroTerminatedFloats'],
-  7: ['optional/content', 'optional/format']
+  7: [
+    'format/idn-email',
```

Comment:
Should these be 'optional/format/idn-email' etc., or is it just doing a suffix match?  It's mildly confusing for them not to match 'optional/content' etc. but I don't feel strongly about it.

Reply:
Yes, that's how the library names the test groups - it uses the two last segments of the path... Just fixed it to match the names with "-".

Reply:
Oh, cool, I had not realized that.  That's totally fine then, thanks!